src/java/fr/paris/lutece/plugins/identitymediation/web/IdentityDuplicateJspBean.java

```diff
@@ -639,13 +639,16 @@ public String doCreateIdentityMergeTask(final HttpServletRequest request) throws
         if(!_canNotify) {
             throw new AccessDeniedException("You don't have the right to create a notification task");
         }
+        if(!_ruleAllowingNotify) {
```

Comment:
should be something like : 
( firstConnected && secondConnected && !_ruleAllowingNotifu )